stack.yaml

```diff
@@ -131,3 +142,11 @@ ghc-options:
   cardano-sl:            -Werror
   cardano-sl-tools:      -Werror
   cardano-sl-lwallet:    -Werror
+  cardano-sl-explorer:   -Werror
+
+# Added for explorer
+flags:
+  cardano-sl:
+    with-explorer: True
```

Comment:
@akegalj No, I would prefer if we all agreed on what is it that we need to do here first, seems we each have a different point of view.
I still think that @domenkozar wanted to have a _explorer_ build in Stack, since we asked me to merge the two Stack files. Maybe he can clarify?